express.js

```diff
@@ -0,0 +1,10 @@
+const express=require("express");
+const app=express();
+
+app.get("/ping",(req,res)=>{
```

Comment:
It's recommended to add error handling middleware to handle any errors that might occur during the execution of your application. [important]